web/server/api/endpoints/lineage.py

```diff
@@ -66,7 +56,6 @@ async def column_lineage(
             sources={
                 model: context.models[model].render_query()
                 for model in context.dag.upstream(model_name)
-                if model in context.models
```

Comment:
This was added to deal with external tables that will appear in the upstream DAG but are not models so their queries cannot be rendered. Is it causing an issue?

Reply:
If something is not a model, it should default to be a CTE node 

![Screenshot 2023-05-30 at 2 28 39 PM](https://github.com/TobikoData/sqlmesh/assets/5644753/10a9ea24-40d9-45a7-a395-6ddbd1d7cc3a)

Reply:
oh, i see, yeah need to bring it back,

Reply:
A CTE wouldn't show up in in a model's upstream DAG though. This only handles external models.